macros/edr/data_monitoring/monitors_query/table_monitoring_query.sql

```diff
@@ -263,7 +263,7 @@
 {% else %}
     select
         {{ elementary.const_as_string('event_freshness') }} as metric_name,
-        {{ elementary.timediff('second', elementary.cast_as_timestamp("max({})".format(event_timestamp_column)), elementary.current_timestamp_column()) }} as metric_value
+        {{ elementary.timediff('second', elementary.cast_as_timestamp("max({})".format(event_timestamp_column)), elementary.quote(elementary.get_run_started_at())) }} as metric_value
```

Comment:
Wasn't really a bug, but using `get_run_started_at` makes it more testable as I can override it with `custom_run_started_at`.